src/dotnet/AzureAI.Proxy/AzureAI.Proxy.csproj

```diff
@@ -25,6 +26,12 @@
     <PackageReference Include="Yarp.ReverseProxy" Version="2.1.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Content Update="proxyconfig.json">
```

Comment:
Do we always need this file, or do we first look for the app config where this can be stored as well.

Reply:
we don't need that file, I've put it in .gitignore, but will remove completely